setup.py

```diff
@@ -352,7 +350,10 @@ def run_tests(self):
         extras_require=EXTRAS_REQUIRE,
         entry_points={
             'hotdoc.extensions': ('get_extension_classes = '
-                                  'hotdoc.extensions:get_extension_classes')},
+                                  'hotdoc.extensions:get_extension_classes'),
+            'console_scripts': [
+                'hotdoc=hotdoc.run_hotdoc:_main',
```

Comment:
Any reason for having the underscore prefix to main here but not in the other script? Just curious, I don't really mind either way :)

Reply:
There was already an underscore on `_main` in that file, so I kept it the way it was. I can remove it for consistency if you prefer.